types/cannon/index.d.ts

```diff
@@ -454,6 +454,15 @@ declare module CANNON {
         type?: number;
         linearDamping?: number;
         angularDamping?: number;
+        allowSleep?: boolean;
+        sleepSpeedLimit?: number;
+        sleepTimeLimit?: number;
+        collisionFilterGroup?: number;
+        collisionFilterMask?: number;
+        fixedRotation?: boolean;
+        linearFactor?: Vec3;
```

Comment:
I do not see this property used in [the source code for Body](http://schteppe.github.io/cannon.js/docs/files/src_objects_Body.js.html#l12).

The same goes for `angularFactory`. Where do they come from?

Reply:
Good catch! I've removed `linearFactor` and `angularFactor` from this PR. They are present in the master branch of canon [1], but not in the last release pushed to npm, 0.6.2.

[1] https://github.com/schteppe/cannon.js/blob/master/src/objects/Body.js#L17-L35